LibGAP.jl/test/runtests_includes.jl

```diff
@@ -0,0 +1,6 @@
+
+@test LibGAP.CSTR_STRING(GAP.GAPFuncs.String(GAP.GAPFuncs.PROD(2^59,2^59))) == "332306998946228968225951765070086144"
```

Comment:
Bikeshed time: I think we should put such a test in a different file, say `basic.jl` or `convert.jl`. `runtests_includes.jl` should then just include those files.

The double indirection as opposed to the standard Julia testing is because we autogenerate `runtests.jl`

Reply:
I propose something else: let's put the autogenerated data into a seprate file, say `gap_path.jl`, which then is included by a static, non generated `runtests.jl`